src/components/button.tsx

```diff
@@ -0,0 +1,25 @@
+import Link from "next/link";
```

Comment:
we should add 'use client' here for the on click?

Reply:
might need an explaination for this one

Reply:
nevermind did a google